### PR TITLE
setup validation for tambah reservasi form

### PR DESCRIPTION
**Pull request by rizfirman**
## Related
[CMS Sribaduga - Fungsi Tambah Sesi oleh Operator](https://app.clickup.com/t/86cuz4vu9)

## Changes
Setup validation for tambah reservasi form

## Evidence
title: Setup validation for tambah reservasi form
project: Jabar Super Apps
participants: @rizfirman @Ibwedagama @marsellavaleria19 @agunghide @azhargraha

### CODE REVIEW

File: components/Sribaduga/DialogTambahReservasi/index.vue [hunk 1]
```diff
@@ -152,7 +152,10 @@
                   </div>
                 </div>
                 <div class="mt-4 flex items-center justify-between font-lato">
-                  <p class="text-[14px] font-[400]">Total Tiket Dipesan</p>
+                  <div>
+                    <p class="text-[14px] font-[400]">Total Tiket Dipesan</p>
+                    <small class="text-red-600">{{ errorTicket }}</small>
```
Comment (azhargraha) [inline components/Sribaduga/DialogTambahReservasi/index.vue:157]: i think it's better to put this tag in a `<p></p>` tag, wdyt?

File: components/Sribaduga/DialogTambahReservasi/index.vue [hunk 2]
```diff
@@ -161,102 +164,181 @@
...
-                  />
-                </div>
+                      <input
+                        id="no-tlp"
+                        v-model="phoneNumber"
+                        name="Telepon"
+                        class="form-input block h-[35px] w-full appearance-none rounded-lg border-2 border-gray-300 bg-white px-4 pl-12 text-gray-500 placeholder-gray-400 focus:outline-none"
+                      />
+                    </div>
+                    <small class="text-red-600">{{ errors[0] }}</small>
```
Comment (azhargraha) [inline components/Sribaduga/DialogTambahReservasi/index.vue:223]: this one too

File: components/Sribaduga/DialogTambahReservasi/index.vue [hunk 3]
```diff
@@ -161,102 +164,181 @@
...
+                    name="Alamat instansi"
+                    tag="div"
+                  >
+                    <textarea
+                      id="alamat-instansi"
+                      v-model="instanceAddress"
+                      name="Alamat instansi"
+                      class="w-full rounded-lg border-2 border-gray-300"
+                    />
+                    <small class="text-red-600">{{ errors[0] }}</small>
```
Comment (azhargraha) [inline components/Sribaduga/DialogTambahReservasi/index.vue:339]: and this too

File: components/Sribaduga/DialogTambahReservasi/index.vue (hunk 1 above)
Comment (azhargraha) [inline components/Sribaduga/DialogTambahReservasi/index.vue:157]: i think it's better to put this tag in a `<p></p>` tag, wdyt?

Comment (azhargraha) [inline components/Sribaduga/DialogTambahReservasi/index.vue:157]: i think it's better to put this  `<small></small>` tag in a `<p></p>` tag, wdyt?

File: components/Sribaduga/DialogTambahReservasi/index.vue [hunk 4]
```diff
@@ -161,102 +164,181 @@
...
-                </label>
-                <div
-                  class="relative mb-4 block text-gray-400 focus-within:text-gray-600"
-                >
-                  <div
-                    class="pointer-events-none absolute top-1/2 flex h-[35px] w-10 -translate-y-1/2 transform items-center justify-center rounded-l-lg border-2 bg-[#F5F5F5]"
+                <ValidationObserver ref="form_informasi_pemesan" tag="div">
+                  <p class="mb-4 font-lato text-[16px] font-[700]">
+                    Informasi Pemesan
+                  </p>
```
Comment (azhargraha) [inline components/Sribaduga/DialogTambahReservasi/index.vue:170]: test

**Review by azhargraha — APPROVED** [review]

**Review by marsellavaleria19 — APPROVED** [review]

**Review by Ibwedagama — APPROVED** [review]

**Review by agunghide — APPROVED** [review]

**Pull request merged**